Jenkinsfile

```diff
@@ -37,43 +23,54 @@ pipeline {
         sh 'sudo /usr/bin/hab-pkg-upload $(ls -rt results/*.hart | head -n 1)'
 
         script {
+            def poolHost = env.RET_DARK_POOL_HOST
+            def slackURL = env.SLACK_URL
+            def buildNumber = env.BUILD_NUMBER
+            def jobName = env.JOB_NAME
+            def onlyPromoteToStage = env.ONLY_PROMOTE_TO_STAGE
+            def stageChannel = env.STAGE_CHANNEL
+
             // Grab IDENT file and cat it from .hart
             def s = $/eval 'ls -rt results/*.hart | head -n 1'/$
```

Comment:
I think this line grabs the _oldest_ `.hart` file whereas I would expect it to grab the _newest_ one. I don't think it matters because we clear `/results` in the `prebuild` step -- but might be worth fixing anyway.

```
 def s = $/eval 'ls -t results/*.hart | head -n 1'/$
 ```